### PR TITLE
Feat/add authorizer elements in handler event

### PR DESCRIPTION
**Pull request by Sc0ra**

### CODE REVIEW

File: packages/serverless-contracts/package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.18.3",
+    "aws-lambda": "^1.0.7",
```
Comment (fargito) [inline packages/serverless-contracts/package.json:42]: We shouldn't need this. I tried https://github.com/swarmion/swarmion/pull/147, can you rebase on it when it is merged?

File: packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { ApiGatewayContract } from '../apiGatewayContract';
 import { OutputType, RequestArguments } from '../types';
 import { getRequestParameters } from './requestParameters';
 
+// Naming cassé
+// Polymorphisme pour gérer axios et fetch dans la même "feature"
```
Comment (fargito) [inline packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts:8]: Shouldn't this be in English?

File: packages/serverless-contracts/src/contracts/apiGateway/types/lambdaHandler.ts [hunk 3]
```diff
@@ -1,17 +1,21 @@
+import { APIGatewayProxyWithCognitoAuthorizerEvent } from 'aws-lambda';
```
Comment (fargito) [inline packages/serverless-contracts/src/contracts/apiGateway/types/lambdaHandler.ts:1]:
```suggestion
import type { APIGatewayProxyWithCognitoAuthorizerEvent } from 'aws-lambda';
```
This should allow to skip `aws-lambda` as a dependency

File: packages/serverless-contracts/package.json (hunk 1 above)
Comment (Sc0ra) [reply packages/serverless-contracts/package.json:42]: i need to do the same in this package but yes i will try it

File: packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts (hunk 2 above)
Comment (Sc0ra) [reply packages/serverless-contracts/src/contracts/apiGateway/features/axiosRequest.ts:8]: it should not have been merged, conversation for a next PR, i'll remove it

**Review by fargito — APPROVED** [review]

**Review by fargito — APPROVED** [review]

**Comment by fargito** [comment]
Great! Thanks a lot!

**Pull request merged**